vendor-extra/JatsContentBundle/src/EventListener/BuildView/FrontDisplayChannelContentMetaListener.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\ContextAwareTranslation;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class FrontDisplayChannelContentMetaListener
+{
+    use ContextAwareTranslation;
+    use SimplifiedViewConverterListener;
+
+    private const CATEGORIES_PATH = 'jats:article-meta/jats:article-categories';
+    private const GROUP_PATH = self::CATEGORIES_PATH.'/jats:subj-group[@subj-group-type="display-channel"]';
+    private const SUBJECT_PATH = self::GROUP_PATH.'/jats:subject';
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter, TranslatorInterface $translator)
+    {
+        $this->converter = $converter;
+        $this->translator = $translator;
+    }
+
+    protected function handle(Element $object, View $view) : View
+    {
+        $items = $view->getArgument('items') ?? [];
+
+        if (isset($items['type'])) {
```

Comment:
so this is an alternative to the other Listener, only one of them will be able to add to `$items`. Where is the priority configured?

Reply:
https://github.com/libero/browser/blob/2dfdfda5d68035405db39c8dcf1e3368947d4ac2/vendor-extra/JatsContentBundle/src/Resources/config/services.xml#L92

Reply:
Can also run:

```sh
bin/console debug:event-dispatcher libero.view.build

Registered Listeners for "libero.view.build" Event
==================================================

 ------- -------------------------------------------------------------------------------------------------------- ----------
  Order   Callable                                                                                                 Priority
 ------- -------------------------------------------------------------------------------------------------------- ----------
  #1      Libero\ViewsBundle\EventListener\BuildView\LangListener::onBuildView()                                   100
  #2      Libero\JatsContentBundle\EventListener\BuildView\HeadingListener::onBuildView()                          0
  #3      Libero\JatsContentBundle\EventListener\BuildView\FrontItemTagsListener::onBuildView()                    0
  #4      Libero\JatsContentBundle\EventListener\BuildView\KeywordGroupTagListListener::onBuildView()              0
  #5      Libero\JatsContentBundle\EventListener\BuildView\FrontArticleTitleContentHeaderListener::onBuildView()   0
  #6      Libero\JatsContentBundle\EventListener\BuildView\FrontContentHeaderMetaListener::onBuildView()           0
  #7      Libero\JatsContentBundle\EventListener\BuildView\FrontDisplayChannelContentMetaListener::onBuildView()   0
  #8      Libero\JatsContentBundle\EventListener\BuildView\FrontSubjectGroupContentHeaderListener::onBuildView()   0
  #9      Libero\JatsContentBundle\EventListener\BuildView\ItalicListener::onBuildView()                           0
  #10     Libero\JatsContentBundle\EventListener\BuildView\BoldListener::onBuildView()                             0
  #11     Libero\JatsContentBundle\EventListener\BuildView\ParagraphListener::onBuildView()                        0
  #12     Libero\JatsContentBundle\EventListener\BuildView\SectionListener::onBuildView()                          0
  #13     Libero\JatsContentBundle\EventListener\BuildView\SubListener::onBuildView()                              0
  #14     Libero\JatsContentBundle\EventListener\BuildView\SupListener::onBuildView()                              0
  #15     Libero\LiberoContentBundle\EventListener\BuildView\FrontContentHeaderListener::onBuildView()             0
  #16     Libero\LiberoContentBundle\EventListener\BuildView\TitleHeadingListener::onBuildView()                   0
  #17     Libero\LiberoContentBundle\EventListener\BuildView\ItalicListener::onBuildView()                         0
  #18     Libero\LiberoContentBundle\EventListener\BuildView\BoldListener::onBuildView()                           0
  #19     Libero\LiberoContentBundle\EventListener\BuildView\SubListener::onBuildView()                            0
  #20     Libero\LiberoContentBundle\EventListener\BuildView\SupListener::onBuildView()                            0
  #21     Libero\JatsContentBundle\EventListener\BuildView\LinkListener::onBuildView()                             -10
  #22     Libero\JatsContentBundle\EventListener\BuildView\FrontArticleTypeContentMetaListener::onBuildView()      -100
 ------- -------------------------------------------------------------------------------------------------------- ----------
```

Reply:
(I assume there is a way to override that file)